CHANGES.md

```diff
@@ -1,3 +1,20 @@
+## edge-19.5.3
+
+* CLI
+  * **New** Added a `linkerd edges` command that shows TLS status
+    between src and dst resources
+  * Tap can now be disabled by using the `--disable-tap` flag
```

Comment:
I would be more specific on this, saying it can be disabled for specific pods during injection by ways of the `--disable-tap`  flag, or its corresponding `config.linkerd.io/disable-tap` annotation.